lib/puppet/provider/service/systemd.rb

```diff
@@ -1,5 +1,7 @@
 # Manage systemd services using systemctl
 
+require 'puppet/file_system/file_impl'
```

Comment:
probably just want to `require 'puppet/file_system'` and let it figure out which impl to require.

Reply:
Done.